Makefile

```diff
@@ -24,6 +24,8 @@ target = kuma
 requirements = -r requirements/local.txt
 # set Django settings module if not already set as env var
 export DJANGO_SETTINGS_MODULE ?= kuma.settings.testing
+export KUMA_REVISION_HASH ?= $(shell git rev-parse HEAD)
+export KUMASCRIPT_REVISION_HASH ?= $(shell cd kumascript && git rev-parse HEAD)
```

Comment:
``git`` is not installed inside the ``web`` container, and running ``make build-static`` or other make commands outputs:

```
make: git: Command not found
/bin/sh: 1: git: not found
```

Please avoid it by directing these messages to ``/dev/null``, like:

```
export KUMA_REVISION_HASH ?= $(shell git rev-parse HEAD 2>/dev/null)
export KUMASCRIPT_REVISION_HASH ?= $(shell cd kumascript && git rev-parse HEAD 2>/dev/null)
```

Reply:
Good catch, thanks!